### PR TITLE
fix: hardcode "en" as default language for llama-parse and use llama cloud key from env

### PR DESCRIPTION
**Pull request by leehuwuj**

### CODE REVIEW

File: packages/create-llama/helpers/index.ts [hunk 1]
```diff
@@ -95,22 +95,34 @@ const createEnvLocalFile = async (
...
+  llamaCloudKey?: string,
+) {
   if (packageManager) {
     const runGenerate = `${cyan(
       framework === "fastapi"
         ? "poetry run python app/engine/generate.py"
         : `${packageManager} run generate`,
     )}`;
     const hasOpenAiKey = openAiKey || process.env["OPENAI_API_KEY"];
+    const hasLlamaCloudKey = (dataSource?.config as FileSourceConfig)
```
Comment (leehuwuj) [inline packages/create-llama/helpers/index.ts:114]: I've considered naming it `checkLlamaCloudKey`, but `hasLlamaCloudKey` looks better in this context.

File: packages/create-llama/helpers/index.ts [hunk 2]
```diff
@@ -95,22 +95,34 @@ const createEnvLocalFile = async (
...
+      ?.useLlamaParse
+      ? llamaCloudKey || process.env["LLAMA_CLOUD_API_KEY"]
+      : true;
     const hasVectorDb = vectorDb && vectorDb !== "none";
     if (framework === "fastapi") {
-      if (hasOpenAiKey && !hasVectorDb && isHavingPoetryLockFile()) {
+      if (
+        hasOpenAiKey &&
+        hasLlamaCloudKey &&
```
Comment (marcusschiesser) [inline packages/create-llama/helpers/index.ts:122]: how about naming it like this?
```suggestion
        openAiKeyConfigured &&
        llamaCloudKeyConfigured &&
```

File: packages/create-llama/questions.ts [hunk 3]
```diff
@@ -605,11 +612,8 @@ export const askQuestions = async (
           {
             type: "text",
             name: "llamaCloudKey",
-            message: "Please provide your LlamaIndex Cloud API key:",
-            validate: (value) =>
-              value
-                ? true
-                : "LlamaIndex Cloud API key is required. You can get it from: https://cloud.llamaindex.ai/api-key",
+            message:
+              "Please provide your LlamaIndex Cloud API key (leave blank to skip):",
```
Comment (marcusschiesser) [inline packages/create-llama/questions.ts:616]: do we add `https://cloud.llamaindex.ai/api-key` to the .env file?

File: packages/create-llama/questions.ts [hunk 4]
```diff
@@ -222,11 +222,18 @@ export const askQuestions = async (
         ];
 
         const hasOpenAiKey = program.openAiKey || process.env["OPENAI_API_KEY"];
+        // If using LlamaParse, require LlamaCloud API key
+        const hasLlamaCloudKey = (
```
Comment (marcusschiesser) [inline packages/create-llama/questions.ts:226]: if you decide to change the var names, here too please

File: packages/create-llama/helpers/index.ts (hunk 2 above)
Comment (leehuwuj) [reply packages/create-llama/helpers/index.ts:122]: It's nice, thanks for the suggestion. I'm updating it, naming is hard :D

File: packages/create-llama/questions.ts (hunk 3 above)
Comment (leehuwuj) [reply packages/create-llama/questions.ts:617]: I made an update to add a minor config guide to .env file. Now, if user left this option as blank then the .env will look like this:
```
# Please obtain the Llama Cloud API key from https://cloud.llamaindex.ai/api-key 
# and set it to the LLAMA_CLOUD_API_KEY variable below.
# LLAMA_CLOUD_API_KEY=
```

File: packages/create-llama/helpers/index.ts [hunk 5]
```diff
@@ -54,6 +54,10 @@ const createEnvLocalFile = async (
 
   if (opts?.llamaCloudKey) {
     content += `LLAMA_CLOUD_API_KEY=${opts?.llamaCloudKey}\n`;
+  } else {
+    content += `# Please obtain the Llama Cloud API key from https://cloud.llamaindex.ai/api-key
```
Comment (marcusschiesser) [inline packages/create-llama/helpers/index.ts:58]: this would copy this content all the time - even if using llamacloud is not configured, right?

Comment (leehuwuj) [reply packages/create-llama/helpers/index.ts]: Thanks, it's a good catch.

**Review by marcusschiesser — APPROVED** [review]

**Pull request merged**